po/fr.po

```diff
@@ -1,5 +1,723 @@
+msgstr "Transparence :"
+
+#: src/Dialogs/ExportDialog.vala:218
+msgid "Scale:"
+msgstr "Échelle :"
+
+#: src/Dialogs/ExportDialog.vala:242 src/FileFormat/FileManager.vala:49
+#: src/FileFormat/FileManager.vala:105
+msgid "Cancel"
+msgstr "Annuler"
+
+#: src/Dialogs/ExportDialog.vala:249 src/Dialogs/ShortcutsDialog.vala:63
+#: src/Layouts/HeaderBar.vala:112
+msgid "Export"
+msgstr "Exporter"
+
+#: src/Dialogs/ReleaseDialog.vala:47 src/Dialogs/SettingsDialog.vala:217
+msgid ""
+"WARNING!\n"
+"Akira is still under development and not ready for production. Missing "
+"features, random bugs, and black holes opening in your kitchen are to be "
+"expected."
+msgstr ""
+"ATTENTION !\n"
+"Akira est encore en cours de développement et n'est pas prêt à être mis en "
+"production. Il faut s'attendre à ce que des fonctionnalités soit "
+"manquantes, qu'il y ait des bugs aléatoires et que des trous noirs "
+"s'ouvrent dans votre cuisine."
+
+#: src/Dialogs/ReleaseDialog.vala:100 src/Dialogs/SettingsDialog.vala:249
+msgid "Make a Donation"
+msgstr "Faire un don"
+
+#: src/Dialogs/ReleaseDialog.vala:109 src/Dialogs/SettingsDialog.vala:258
+msgid "Suggest Translations"
+msgstr "Suggérer des traductions"
+
+#: src/Dialogs/ReleaseDialog.vala:118 src/Dialogs/SettingsDialog.vala:267
+msgid "Report a Problem"
+msgstr "Signaler un problème"
+
+#: src/Dialogs/SettingsDialog.vala:40
+msgid "Preferences"
+msgstr "Préférences"
+
+#: src/Dialogs/SettingsDialog.vala:50 src/Dialogs/SettingsDialog.vala:73
+#: src/Dialogs/ShortcutsDialog.vala:45
+msgid "General"
+msgstr "Général"
+
+#: src/Dialogs/SettingsDialog.vala:51 src/Dialogs/SettingsDialog.vala:86
+msgid "Interface"
+msgstr "Interface"
+
+#: src/Dialogs/SettingsDialog.vala:52 src/Layouts/HeaderBar.vala:280
+msgid "Shapes"
+msgstr "Formes"
+
+#: src/Dialogs/SettingsDialog.vala:53
+msgid "About"
+msgstr "À propos"
+
+#: src/Dialogs/SettingsDialog.vala:74
+msgid "Auto Reopen Latest File:"
+msgstr "Rouvrir automatiquement le dernier fichier :"
+
+#: src/Dialogs/SettingsDialog.vala:88
+msgid "Use Dark Theme:"
+msgstr "Utiliser le thème sombre :"
+
+#: src/Dialogs/SettingsDialog.vala:92
+msgid "Invert Panels Order:"
+msgstr "Inverser l'ordre des panneaux :"
+
+#: src/Dialogs/SettingsDialog.vala:96
+msgid "Restart application to apply this change."
+msgstr "Redémarrer l'application pour appliquer ce changement."
+
+#: src/Dialogs/SettingsDialog.vala:100
+msgid "ToolBar Style"
+msgstr "Style de la barre d'outils"
+
+#: src/Dialogs/SettingsDialog.vala:102
+msgid "Show Button Labels:"
+msgstr "Afficher l'étiquette des boutons :"
+
+#: src/Dialogs/SettingsDialog.vala:106
+msgid "Use Symbolic Icons:"
+msgstr "Utiliser les icônes symboliques :"
+
+#: src/Dialogs/SettingsDialog.vala:129
+msgid "Default Colors"
+msgstr "Couleurs par défaut"
+
+#: src/Dialogs/SettingsDialog.vala:131
+msgid "Define the default style used when creating a new shape."
+msgstr ""
+"Définissez le style par défaut utilisé lors de la création d'une nouvelle "
```

Comment:
If possible I would like to have some context for this one to be sure it's the right conjugation that is used.

Reply:
It's a configuration:
![image](https://user-images.githubusercontent.com/22885898/95777746-0eb99280-0cc7-11eb-9420-1d35dc4cb059.png)

Reply:
In this case I would rather say 
> Définit le style par défaut utilisé lors de la création d'une nouvelle forme.